### PR TITLE
Plugin Changelog for local plugins

### PR DESCRIPTION
**Pull request by shyim**
## Description

| Questions               | Answers |
|-------------------------|-------------------------------------------------------|
| Why?                    | Local plugins should have the same abilities like in store |
| BC breaks?              | no |
| Tests exists & pass?    | yes |
| Related tickets?        |  |
| How to test?            | Install latest version of https://github.com/shyim/shopware-profiler from github |
| Requirements met?       | Yep |

Screenshot:
image

plugin.xml
```xml
<?xml version="1.0" encoding="utf-8"?>
<plugin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="../../../engine/Shopware/Components/Plugin/schema/plugin.xsd">
    <label lang="de">ShyimProfiler</label>
    <label lang="en">ShyimProfiler</label>
    <version>1.1.0</version>
    <copyright>Shyim</copyright>
    <license>MIT</license>
    <link>https://github.com/shyim/shopware-profiler</link>
    <author>Shyim</author>
    <changelog version="1.1.0">
        <changes lang="en">
            <![CDATA[
                <h4>Performance Event Charts</h4>
                <ul>
                <li>Added event charts to show which events needs how long to execute</li>
                <li>Added overview of cache read/write/delete and a hit/miss metric</li>
                <li>Added new sidebar of security with all user informations and password encoders</li>
                <li>Added overview of container parameters to "Configuration" tab</li>
                <li>Added search form to developers.shopware.com</li>
                </ul>
            ]]>
        </changes>
    </changelog>
</plugin>
```

### CODE REVIEW

File: _sql/migrations/943-add-plugin-changelog.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class Migrations_Migration943 extends Shopware\Components\Migrations\AbstractMigration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function up($modus)
+    {
+        $sql = <<<'SQL'
+ALTER TABLE `s_core_plugins` ADD `changelog` TEXT CHARACTER SET utf8 COLLATE utf8_unicode_ci NULL DEFAULT NULL AFTER `description`;
```
Comment (mitelg) [inline _sql/migrations/943-add-plugin-changelog.php]: there is already a column `changes` 
I think it was created with intention to store the changelog. maybe you can use this?

**Review by mitelg — CHANGES_REQUESTED** [review]

File: _sql/migrations/943-add-plugin-changelog.php (hunk 1 above)
Comment (shyim) [reply _sql/migrations/943-add-plugin-changelog.php]: I guess I was drunken. I will change it later

Comment (mitelg) [reply _sql/migrations/943-add-plugin-changelog.php]: I bet you were not :wink: :smile:

Comment (shyim) [reply _sql/migrations/943-add-plugin-changelog.php]: @mitelg Captain fixed!

**Review by mitelg — APPROVED** [review]

**Comment by soebbing** [comment]
Hi @shyim, we just merged your PR: https://github.com/shopware/shopware/commit/79856faf0fba0a5c80770a498597840d6a0b1e8f (and added a bit of HTML sanitizing). 😉 

 Thank you again for your awesome help! 💙

**Pull request closed without merging**